src/arch/riscv/linux/fs_workload.cc

```diff
@@ -75,6 +78,39 @@ FsLinux::initState()
     }
 }
 
+void
+FsLinux::startup()
+{
+    KernelWorkload::startup();
+
+    addExitOnKernelPanicEvent();
+    addExitOnKernelOopsEvent();
+}
+
+void
+FsLinux::addExitOnKernelPanicEvent()
+{
+    const std::string dmesg_output = name() + ".dmesg";
+    if (params().exit_on_kernel_panic) {
+        kernelPanicPcEvent = addFuncEventOrPanic<linux::KernelPanic>(
```

Comment:
Use a different version of this function so that it doesn't panic if it can't find the symbol. Instead, we want to have a warning.